### PR TITLE
Fixed internal module issue

### PR DESCRIPTION
**Pull request by jasondown**
PlaceOrderImplementation types need to be public since they are exposed via PlaceOrder.Api. The actual implementation details are still internal via a submodule.

### CODE REVIEW

**Comment by jasondown** [comment]
I'm not sure how to link this to issue#13 where this was first reported.

File: src/OrderTaking/PlaceOrder.Implementation.fs [hunk 1]
```diff
@@ -121,315 +121,315 @@ type CreateEvents =
 // ======================================================
 // Section 2 : Implementation
 // ======================================================
+module internal Implementation =
```
Comment (jasondown) [inline src/OrderTaking/PlaceOrder.Implementation.fs:124]: Submodule created that keeps implementation internal. All changes below this are just due to indentation (no real code changes).

File: src/OrderTaking/PlaceOrder.Implementation.fs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿module internal OrderTaking.PlaceOrder.Implementation
+﻿module OrderTaking.PlaceOrder.ImplementationTypes
```
Comment (jasondown) [inline src/OrderTaking/PlaceOrder.Implementation.fs:1]: Changed this module to be public, containing only the types that are exposed in the PlaceOrder Api module (hence the name change). A submodule was created below that keeps the implementation section internal.

**Comment by swlaschin** [comment]
Thanks again -- comments in #13

**Pull request closed without merging**